.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": ".NET Core Launch (console)",
+            "type": "coreclr",
+            "request": "launch",
+            "program": "${workspaceRoot}/tools/Cake.CoreCLR/Cake.dll",
+            "args": [
+                "${workspaceRoot}/build.cake",
+                "--debug",
+                "--verbosity=diagnostic"
+            ],
+            "cwd": "${workspaceRoot}",
+            "stopAtEntry": true,
+            "externalConsole": false
+        }
+    ]
+}
```

Comment:
@gep13, @pascalberger: Okay, good to know. I agree that it should be in the `.gitignore` just to make it explicit. 🙂

The `set_PriorityClass` problem is hopefully fixed by GitTools/GitTools.Core#38, but we need to get that released somehow. Kind of related to this PR, but for different reasons and another repository. 😄